core/Seeder.php

```diff
@@ -1,113 +1,131 @@
 <?php
 /**************************************************************************
- MantisBT Seeder Plugin
- Copyright (c) MantisHub - Victor Boctor
- All rights reserved.
- MIT License
- **************************************************************************/
+	* MantisBT Seeder Plugin
```

Comment:
There is a lot of noise in the indentation, can you please update it to follow the convention of the original code of the plugin to remove the noise and be consistent.